### DISCUSSION TITLE
Can't find LinearTransition

### DISCUSSION THREAD

**Discussion by mrAlphak** [software-mansion/react-native-reanimated#5633] [Q&A] [answered]
Hello, i'm currently building a custom toast with react native and i use reanimated for my animations, i want to make a smooth transition between each toast, but i can't find the "linear transition" from reanimated, however, it is clearly indicated in the documentation.

![image](https://github.com/software-mansion/react-native-reanimated/assets/35864484/4b79c36a-5cc0-4262-a5fc-a1c72fa8b436)

![image](https://github.com/software-mansion/react-native-reanimated/assets/35864484/227d7f1b-61ac-4e74-bf03-944497a904cc)

Reanimated version: 3.3.0
React native version: 0.72.6

**Comment by MatiPl01** [comment, marked as answer]
To use `LinearTransition` you would have to upgrade `react-native-reanimated`. It works fine on the latest version.
